src/main/java/com/salesforce/dynamodbv2/mt/util/DynamoDbCapacity.java

```diff
@@ -60,12 +61,8 @@ public static void setBillingMode(CreateTableRequest createTableRequest, Billing
             createTableRequest.withBillingMode(BillingMode.PAY_PER_REQUEST);
         } else if ((billingModeFromRequest == null || billingModeFromRequest.equals(BillingMode.PROVISIONED.toString()))
                 && createTableRequest.getProvisionedThroughput() == null) {
-
-            if (createTableRequest.getProvisionedThroughput() == null) {
-                createTableRequest.withProvisionedThroughput(new ProvisionedThroughput(
-                        1L, 1L));
-            }
-
+            createTableRequest.withProvisionedThroughput(new ProvisionedThroughput(
+                    1L, 1L));
```

Comment:
@degaydorus ... This was caught by "Condition is always 'true'" ... is it ok?

Reply:
@msgroi having trouble reproducing this locally with the master branch code (verified DynamoDbCapacity file).  The line highlighted is: 
```
createTableRequest.withProvisionedThroughput(new ProvisionedThroughput(1L, 1L));
```
what specifically did it say was always true (Since it's just setting a PT value)? the else if block?

Reply:
Thanks @degaydorus.

The code was ...

```
} else if ((billingModeFromRequest == null || billingModeFromRequest.equals(BillingMode.PROVISIONED.toString()))
                && createTableRequest.getProvisionedThroughput() == null) {

            if (createTableRequest.getProvisionedThroughput() == null) {
                createTableRequest.withProvisionedThroughput(new ProvisionedThroughput(
                        1L, 1L));
            }

            createTableRequest.withBillingMode(BillingMode.PROVISIONED);
        }
```

The problem is that `if (createTableRequest.getProvisionedThroughput() == null) {` always evaluates to `true` because it's nested in an `if` statement that has `&& createTableRequest.getProvisionedThroughput() == null` in it.  

So in this PR, I've removed that inner `if` statement because it's always `true`.  I'm hoping I got this right.

Reply:
yep looks good, thanks